docs/source/serialization.mdx

```diff
@@ -49,255 +60,365 @@ Ready-made configurations include the following models:
 - T5
 - XLM-RoBERTa
 
-This conversion is handled with the PyTorch version of models - it, therefore, requires PyTorch to be installed. If you
-would like to be able to convert from TensorFlow, please let us know by opening an issue.
+The ONNX conversion is supported for the PyTorch versions of the models. If you
```

Comment:
I'm not 100% sure whether we don't support TensorFlow or not since we technically do in the deprecated `convert_graph_to_onnx.py` script

Reply:
I think that it is not well defined for now, we need to work on the export from TF models, but for now I think it is best to keep this.